presto/presto.go

```diff
@@ -603,6 +603,19 @@ func (st *driverStmt) QueryContext(ctx context.Context, args []driver.NamedValue
 		stmt:    st,
 		nextURI: sr.NextURI,
 	}
+	completedChannel := make(chan struct{})
+	defer close(completedChannel)
+	go func() {
+		select {
+		case <-ctx.Done():
+			err := rows.Close()
```

Comment:
The name of the `Close()` method is very confusing, it should've been `Cancel` or something along the lines. Not related to this PR tho...